src/index.ts

```diff
@@ -60,7 +62,7 @@ const getTitle = (
   }
 
   return {
-    title: route?.titleConcat === false ? route.title : titles.reverse().join(` ${divider} `),
+    title: concatenateTitles ? titles.reverse().join(` ${divider} `) : route?.title,
```

Comment:
If `route` is `undefined` it will be false, so it is trying to use `route.title` which is the wrong behavior in this case. That was the reason why I changed the order of the if clause in the recent release. Using a null-check for `route?.title` is a hacky workaround.